datanode/src/storage_api_test.py

```diff
@@ -113,17 +113,17 @@ def testSlippyConversionWithInvalidData(self):
     result = self.app.get('/slippy/11?coords=1,1,2')
     self.assertEqual(400, result.status_code)
 
-  def testSlippyConversionWithValidData(self):
+  def testSlippyConversionWithValidPoints(self):
     r = self.app.get('/slippy/11?coords=1,1')
-    self.assertEqual(r.status_code, 200)
+    self.assertEqual(200, r.status_code)
     j = json.loads(r.data)
     self.assertEqual(11, j['data']['grid_cells'][0]['zoom'])
     self.assertEqual(1029, j['data']['grid_cells'][0]['x'])
     self.assertEqual(1018, j['data']['grid_cells'][0]['y'])
     self.assertEqual('http://tile.openstreetmap.org/11/1029/1018.png',
                      j['data']['grid_cells'][0]['link'])
-    r = self.app.get('/slippy/10?coords=37.408959,-122.053834')
```

Comment:
nit: might want to also keep this test for backwards compatibility checking

Reply:
Test for those lat/lon is still there, just with the point parameter. The non "point" parameter is tested above as well.